examples/text/plot_document_classification_20newsgroups.py

```diff
@@ -171,127 +295,105 @@ def benchmark(clf):
     if hasattr(clf, "coef_"):
         print("dimensionality: %d" % clf.coef_.shape[1])
         print("density: %f" % density(clf.coef_))
-
-        if feature_names is not None:
-            print("top 10 keywords per class:")
-            for i, label in enumerate(target_names):
-                top10 = np.argsort(clf.coef_[i])[-10:]
-                print(trim("%s: %s" % (label, " ".join(feature_names[top10]))))
         print()
 
     print("classification report:")
     print(metrics.classification_report(y_test, pred, target_names=target_names))
 
```

Comment:
I find it weird to have such long printed outputs in a notebook-style example.

I would rather just cut the classification reports out:

```suggestion
```